codes.merritt.FeelingFinder.yml

```diff
@@ -0,0 +1,33 @@
+---
+app-id: codes.merritt.FeelingFinder
+runtime: org.gnome.Platform
+runtime-version: "41"
+sdk: org.gnome.Sdk
+command: feeling_finder
+separate-locales: false
+finish-args:
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--device=dri"
+  - "--socket=pulseaudio"
+  - "--share=network"
+modules:
+  - name: feeling_finder
+    buildsystem: simple
+    build-commands:
+      - "cp -r packaging/linux/flatpak/build-flatpak.sh ."
+      - "cp -r packaging/linux/flatpak/codes.merritt.FeelingFinder.metainfo.xml ."
+      - "cp -r packaging/linux/codes.merritt.FeelingFinder.desktop ."
+      - "cp -r assets/images/icon.svg ."
+      - "cp -r packaging/linux/flatpak/build-flatpak.sh ."
+      - "./build-flatpak.sh"
+    sources:
+      - type: file
+        url: https://github.com/Merrit/feeling_finder/releases/latest/download/FeelingFinder-Linux-Portable.tar.gz
```

Comment:
this is not resolved.

Reply:
see https://github.com/flathub/flathub/wiki/App-Maintenance#only-build-on-x86_64

Reply:
Also the build fail because "latest" is probably pointing to something that change all the time.

You should point to a fixed revision like https://github.com/Merrit/feeling_finder/releases/download/v0.2.0/FeelingFinder-Linux-Portable.tar.gz